src/http/api/views/account.follows.view.ts

```diff
@@ -196,45 +322,14 @@ export class AccountFollowsView {
                 }
 
                 const followeeAccount = await this.db('accounts')
-                    .where('ap_id', actor.id?.toString() || '')
+                    .where('ap_id', followsActorObj.id?.toString() || '')
```

Comment:
This is same as above re: `ap_id_hash`

Reply:
Done in https://github.com/TryGhost/ActivityPub/pull/557/commits/92b298171accb7c5f0b0ab5cc240b5ddff22d18c